analyzer/api.go

```diff
@@ -119,7 +116,7 @@ const (
 	// EventGovernanceVote is a proposal vote event.
 	EventGovernanceVote
 	// EventUnknown is an unknown event type.
-	EventUnknown
+	EventUnknown = 255
```

Comment:
I guess we are going unknown at the end?

Reply:
:+1: this will also prevent new enum values changing `EventUnknown`